tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/graphics/ImageWin32Tests.java

```diff
@@ -37,19 +37,15 @@ public void setUp() {
 	}
 
 	@Test
-	public void imageMustBeRescaledOnZoomChange() {
+	public void imageShouldHaveDimesionAsPerZoomLevel() {
 		int zoom = DPIUtil.getDeviceZoom();
 		Image image = new Image(display, 10, 10);
 
 		try {
 			ImageData baseImageData = image.getImageData(zoom);
 			assertEquals("Width should equal the initial width on the same zoom", 10, baseImageData.width);
-			Image scaledImage = Image.win32_new(image, zoom);
-			ImageData scaledImageData = scaledImage.getImageData(zoom*2);
+			ImageData scaledImageData = image.getImageData(zoom*2);
```

Comment:
Extract the "magic number" `2` to a variable called `scalingFactor` or something similar.

Reply:
Done